pkgs/tools/networking/cdpr/default.nix

```diff
@@ -9,6 +9,10 @@ stdenv.mkDerivation rec {
     sha256 = "1idyvyafkk0ifcbi7mc65b60qia6hpsdb6s66j4ggqp7if6vblrj";
   };
 
+  postPatch = ''
+    substituteInPlace Makefile --replace 'gcc' '"$$CC"'
```

Comment:
Why $$?

Reply:
It's a way to access an env variable inside a makefile. If that raises an eyebrow, an alternative could be `substituteInPlace Makefile --replace 'gcc' '${stdenv.cc.targetPrefix}cc'`, which should evaluate to the same thing.